octoprint_mrbeam/analytics/analytics_handler.py

```diff
@@ -714,6 +714,13 @@ def _write_dust_log(self, data):
 		except Exception as e:
 			self._logger.exception('Error during write dust_log: {}'.format(e.message))
 
+	def _write_fan_rpm_test(self, data):
```

Comment:
seems to be a "public" method, so pls remove the underscore

Reply:
done